jasmine-2.5.2/src/Bonus.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+function Bonus() {
+  this._strikeOrSpare = "";
```

Comment:
The combo of a string and a number tells me there's definitely some concept hiding here. In addition, I'm surprised to see `strikeOrSpare` return a string. I think this could benefit from some sort of `Strike` or `Spare` object, perhaps.